samples/sample_hevc_fei_abr/include/dso/hevc2_parser.h

```diff
@@ -431,28 +431,28 @@ class SDParser //Slice data parser
         if (std::is_same<CTU, T>::value)
         {
             assert(m_ctu.capacity() >= m_ctu.size() + 1);
-            CTU z = {};
+            CTU z{};
             m_ctu.push_back(z);
```

Comment:
emplace_back(CTU{}) should be a bit better

Reply:
my bad, this one will do move construction, but not perfect forwarding, emplace_back(std::initializer_list<CTU>{}) will do the trick